db/migration/1665135955821-AddRedirectsTablesSQLFile.sql

```diff
@@ -0,0 +1,1773 @@
+-- This file is the SQL only content of the TypeORM migration of the same name.
+-- It exists for review purposes etc and is not intended to be run directly (although
+-- you can of course run fragments of this file against our MySQL database if you like)
```

Comment:
I don't love that we have two files that do the same thing, just one's SQL and one's TS.

Not sure if this will work, but is it an option to (pseudocode incoming):
```
const allSql = await fs.read("AddRedirectsTable.sql")
const sqlArr = split allSql into statements
for sql in sqlArr:
  run sql
```
Or maybe there's also a TypeORM method that can run multiple statements, not sure.

Otherwise, you could consider adding `-- START` and `-- END` comments to facilitate the splitting.